### PR TITLE
fix(ingestion/powerbi): m-query fixes

### PR DESCRIPTION
**Pull request by sid-acryl**
- MS Sql data-access function having multiple arguments 
- remove `drop if exist` clause for comma separated table names

### CODE REVIEW

File: metadata-ingestion/src/datahub/ingestion/source/powerbi/m_query/native_sql_parser.py [hunk 1]
```diff
@@ -72,10 +72,21 @@ def get_tables(native_query: str) -> List[str]:
...
+        #   "DROP TABLE IF EXISTS #<identifier>, <identifier2>, ...\n"
+        r"DROP\s+TABLE\s+IF\s+EXISTS\s+(?:#?\w+(?:,\s*#?\w+)*)[;\n]",
+    ]
+
+    new_query = query
+
+    for pattern in patterns:
+        new_query = re.sub(pattern, "", new_query, flags=re.IGNORECASE)
+
+    return new_query
```
Comment (hsheth2) [inline metadata-ingestion/src/datahub/ingestion/source/powerbi/m_query/native_sql_parser.py:89]: this method needs unit tests

**Review by hsheth2 — CHANGES_REQUESTED** [review]

File: metadata-ingestion/src/datahub/ingestion/source/powerbi/m_query/native_sql_parser.py (hunk 1 above)
Comment (sid-acryl) [reply metadata-ingestion/src/datahub/ingestion/source/powerbi/m_query/native_sql_parser.py:89]: It is tested under `test_unsupported_data_platform`, However will explicit test cases.

**Comment by sid-acryl** [comment]
Closing this pull request as it has some unwanted commit

File: metadata-ingestion/src/datahub/ingestion/source/powerbi/m_query/native_sql_parser.py (hunk 1 above)
Comment (sid-acryl) [reply metadata-ingestion/src/datahub/ingestion/source/powerbi/m_query/native_sql_parser.py:92]: This review comment is addressed in PR: https://github.com/datahub-project/datahub/pull/11906. Test case `test_drop_statement`

**Pull request closed without merging**